src/utils/chart.js

```diff
@@ -228,13 +229,27 @@ export const updateDateBuckets = (name, buckets, areaBuckets) => {
   );
 };
 
-export const externalTooltipFormatter = (tooltip) => {
+export const externalTooltipFormatter = (tooltip, colorMap) => {
   if (!tooltip) {
     return tooltip;
   }
+  const newTooltip = { ...tooltip };
   const parts = tooltip.title.split(':');
+  newTooltip.values = cloneDeep(tooltip.values);
```

Comment:
That was giving an error since tooltip.values is an Array.  I went ahead and switched it over to structuredClone and added a global polyfill in setupTests.js